mender.go

```diff
@@ -357,15 +357,19 @@ func (m *mender) FetchUpdate(url string) (io.ReadCloser, int64, error) {
 // that occurred. If no update is available *UpdateResponse is nil, otherwise it
 // contains update information.
 func (m *mender) CheckUpdate() (*client.UpdateResponse, menderError) {
-	currentArtifactName := m.GetCurrentArtifactName()
-	//TODO: if currentArtifactName == "" {
-	// 	return errors.New("")
-	// }
+	currentArtifactName, err := m.GetCurrentArtifactName()
+	if err != nil {
+		log.Error("could not get the current artifact name")
```

Comment:
A log should be good enough for notifying the user that we could not get the artifact?